src/transpiler.js

```diff
@@ -8,7 +10,12 @@ const transpiler = {
       phone: getPhone(source),
       email: getEmail(source),
       birthday: getBirthday(source),
-      address: getAddress(source)
+      address: getAddress(source),
+      metadata: {
+        version: SCHEMA_VERSION,
+        google: { metadata: source.metadata }
+      },
+      vendorId: source.resourceName
```

Comment:
Shouldn't the `vendorId` be in `metadata`?

Reply:
IDK, what does @ptbrowne think?

Reply:
Yes I think it should be metadata